README.md

```diff
@@ -1,2 +1,165 @@
+[JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples), ([MAUI(Preview)](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples)), and desktop development ([WinForms](https://www.syncfusion.com/winforms-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples), [WPF](https://www.syncfusion.com/wpf-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples), [WinUI](https://www.syncfusion.com/winui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples), [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples), and ([MAUI(Preview)](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=winforms-spreadsheet-github-samples). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+  <a href="mailto:sales@syncfusion.com?Subject=Syncfusion WinForms-Spreadsheet - Visual Studio Marketplace" target="_top">sales@syncfusion.com</a>  | <a href="https://www.syncfusion.com?utm_medium=listing&utm_source=vs_marketplace&utm_campaign=WinForms-Spreadsheet-trial-vs_marketplace">www.syncfusion.com</a>  | Toll Free: 1-888-9 DOTNET <br>
```

Comment:
remove the hyphen and add GitHub instead of Visual Studio Marketplace

<a href="mailto:sales@syncfusion.com?Subject=Syncfusion WinForms Spreadsheet - Github"